### DISCUSSION TITLE
How to render TreeView with dashed lines between nodes

### DISCUSSION THREAD

**Discussion by wgraaf** [AvaloniaUI/Avalonia#12566] [Q&A]
How can the TreeView control be made to show dashed lines between nodes, like the ancient WinForms tree control used to do?
See below:

![image](https://github.com/AvaloniaUI/Avalonia/assets/123667061/65a0f5ec-2ae5-4d3c-9045-7454a681b4f3)

**Comment by timunie** [comment]
I guess similar to WPF solution (heavy lifting)
https://social.msdn.microsoft.com/Forums/en-US/30cb182c-9419-40bd-946e-87971515fb95/show-treeview-nodes-connected-with-dotted-lines?forum=wpf
